recipes/bx/all/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+cmake_minimum_required(VERSION 3.0)
```

Comment:
Where did you find this cmake file? Did you send it to the upstream?

Reply:
It's based on https://github.com/widberg/bgfx.cmake/blob/master/cmake/bx.cmake.

Reply:
but is it not the same right? Did you create it? If yes, I do recommend pushing a PR to the upstream too.

Reply:
It is not the same because the original is hardly linked in the cmake build scripts for bgfx and cannot be used as a standalone file. So, in a way, I "created" it. I can't pushing a PR to the upstream because [the maintainer doesn't want maintain CMake configs](https://github.com/bkaradzic/bgfx/issues/1284#issuecomment-348732490).

Reply:
in this case it seems be okay

Reply:
I would like to have a comment about this in the CMakelists file and add the link as well for future reference

Reply:
I would like to have a comment about this in the CMakelists file and add the link as well for future reference